lib/Mojolicious/Plugin/Yancy.pm

```diff
@@ -471,6 +471,18 @@ This configuration will achieve the same as the above with C<last_updated>:
         },
     }
 
+=head4 yancy.wrap
+
+The configured parameters are a list of strings. For each one, the
```

Comment:
This is another one: Could you add a real-life use-case to the docs to help users understand what they might end up using this for?

Reply:
As noted above, the example is literally from the "realworld" API.

Reply:
I could add to the docs saying it's from the "realworld" thingy if you like?

Reply:
I mean more that "outer" and "inner" are placeholders for real data, and we could just use the real data instead. It's trying to flatten/unflatten data into a relational database, yes? Like I could do:

```
{
    user => {
        name => 'Foo',
        email => 'doug@preaction.me',
    }
}
```

And by doing a `wrap` / `unwrap` pair change it to:

```
{
    name => 'Foo',
    email => 'doug@preaction.me',
}
```

Right?

Reply:
Right! And done :-) (see force-pushed update)